java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/debugging/launch/NbLaunchDelegate.java

```diff
@@ -508,7 +508,7 @@ public void finished(boolean success) {
 
         if (provider == null) {
             command = debug ? mainSource ? ActionProvider.COMMAND_DEBUG
-                                         : ActionProvider.COMMAND_DEBUG // DEBUG_TEST is missing?
+                                         : ActionProvider.COMMAND_TEST //TODO: COMMAND_DEBUG_TEST is missing?
```

Comment:
This change isn't very satisfying, right? Instead of being able to debug something, we run tests with debugging disabled.

Reply:
The specific combination of the method parameters should lead to running of all project tests under a debugger. However, since the `COMMAND_DEBUG_TEST` action is no available yet (it should be implemented in a separate PR) it it IMHO better to run all tests without debugger than to run the project under debugger.